<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>command</class>
 <widget class="QWidget" name="command">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lytCmmnd">
   <property name="geometry">
    <rect>
     <x>-4</x>
     <y>0</y>
     <width>811</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lytCmmnd
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/lytCmmnd.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="mbstRlifBtn">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>52</y>
     <width>148</width>
     <height>93</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#mbstRlifBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/mbstRlifBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="mbnRlifBtn">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>158</y>
     <width>148</width>
     <height>93</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#mbnRlifBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/mbnRlifBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="lvlingRlifBtn">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>265</y>
     <width>148</width>
     <height>93</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lvlingRlifBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/lvlingRlifBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="nrmlRlifModBtn">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>371</y>
     <width>148</width>
     <height>93</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#nrmlRlifModBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/nrmlRlifModBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ctrlSysOnBtn">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>371</y>
     <width>148</width>
     <height>93</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#ctrlSysOnBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/ctrlSysOnBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="stopBtn">
   <property name="geometry">
    <rect>
     <x>328</x>
     <y>371</y>
     <width>148</width>
     <height>93</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#stopBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/stopBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="emergModBtn">
   <property name="geometry">
    <rect>
     <x>483</x>
     <y>371</y>
     <width>148</width>
     <height>93</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#emergModBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/emergModeBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="lvlingStupBtn">
   <property name="geometry">
    <rect>
     <x>643</x>
     <y>265</y>
     <width>148</width>
     <height>93</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lvlingStupBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/lvlingStupBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="mbnStupBtn">
   <property name="geometry">
    <rect>
     <x>643</x>
     <y>158</y>
     <width>148</width>
     <height>93</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#mbnStupBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/mbnStupBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="mbstStupBtn">
   <property name="geometry">
    <rect>
     <x>643</x>
     <y>52</y>
     <width>148</width>
     <height>93</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#mbstStupBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/mbstStupBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="srvicsBtn">
   <property name="geometry">
    <rect>
     <x>197</x>
     <y>239</y>
     <width>163</width>
     <height>105</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#srvicsBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/srvicsBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bckCmmndBtn">
   <property name="geometry">
    <rect>
     <x>444</x>
     <y>239</y>
     <width>163</width>
     <height>105</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#bckCmmndBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/bckCmmndBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="nrmlStupModBtn">
   <property name="geometry">
    <rect>
     <x>643</x>
     <y>371</y>
     <width>148</width>
     <height>93</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#nrmlStupModBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/nrmlStupModBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>96</y>
     <width>401</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#textEdit
{
	background-color :rgb(211, 212, 213);
	border-radius: 15px ;
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Cambria'; font-size:8pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
